### PR TITLE
[WIPTEST] Sentakuification of User/Group

### PR DESCRIPTION
**Pull request by psav**
Sentakuification of User/Group

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4d42b7be84ed22c4297fb155368431103d7e69e1:

`cfme/configure/access_control/__init__.py`:
- :bangbang: Line 130:5: **E303** *too many blank lines (2)*
- :heavy_exclamation_mark: Line 834:22: **W292** *no newline at end of file*

`cfme/configure/access_control/rest.py`:
- :red_circle: Line 27:1: **F811** *redefinition of unused 'create' from line 7*

`cfme/configure/access_control/ui.py`:
- :bangbang: Line 238:1: **E302** *expected 2 blank lines, found 1*
- :red_circle: Line 560:1: **F811** *redefinition of unused 'update' from line 113*
- :red_circle: Line 603:1: **F811** *redefinition of unused 'delete' from line 183*
- :red_circle: Line 651:1: **F811** *redefinition of unused 'edit_tags' from line 222*
- :red_circle: Line 668:1: **F811** *redefinition of unused 'remove_tag' from line 238*
- :red_circle: Line 685:1: **F811** *redefinition of unused 'get_tags' from line 255*
- :red_circle: Line 750:1: **F811** *redefinition of unused 'create' from line 287*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4490a9eca20fa52914df21efbf333f98202fd2c3:

`cfme/tests/configure/test_access_control.py`:
- :red_circle: Line 201:13: **F841** *local variable 'user' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4fad5bd44c9cf319e0ebb1620c9bcf9d1c5a5984:

`cfme/tests/configure/test_access_control.py`:
- :red_circle: Line 130:13: **F841** *local variable 'nu' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 491ff6ebdd9a09b07dffbabbf12a7aff7f564ca4:

`cfme/tests/configure/test_access_control.py`:
- :red_circle: Line 130:13: **F841** *local variable 'nu' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b899b7a7756540c77431613df088f9ef9f8ef9ca:

`cfme/tests/configure/test_access_control.py`:
- :red_circle: Line 130:13: **F841** *local variable 'nu' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 94742890577b415ee805436c81a1581ad3197ed9:

`cfme/tests/configure/test_access_control.py`:
- :red_circle: Line 135:13: **F841** *local variable 'nu' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/configure/access_control/ui.py [hunk 1]
```diff
@@ -0,0 +1,873 @@
...
+        'user_group_select': getattr(
+            updates.get('group'),
+            'description', None)
+    })
+    changed = view.fill({
+        'name_txt': new_updates.get('name_txt'),
+        'userid_txt': new_updates.get('userid_txt'),
+        'password_txt': new_updates.get('password_txt'),
+        'password_verify_txt': new_updates.get('password_verify_txt'),
+        'email_txt': new_updates.get('email_txt'),
```
Comment (mfalesni) [inline cfme/configure/access_control/ui.py]: Can we get rid of the suffixes?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**